traffic.py

```diff
@@ -63,10 +63,15 @@ def send_waku_msg(node_address, kbytes, pubsub_topic, content_topic):
   print("Injecting traffic to single node REST API:", args.single_node)
 
 # this simply converts from http://url_[1..5]:port to
-# [http://url_1:port
+# [http://url_1:port or from http://url-[1..5]:port to
+# [http://url-1:port
 nodes = []
 if args.multiple_nodes:
-  range_nodes = args.multiple_nodes.split(":")[1].split("_")[2]
+  #if args.multiple_nodes contain underscore then split by underscore else split by hyphen
```

Comment:
lgtm. Not required but perhaps we can take the opportunity to also improve it with this. I think this should work in both cases. But at your discretion.

```python
import re
match = re.search(r'\[(\d+)\.\.(\d+)\]', args.multiple_nodes)
if match:
    start = int(match.group(1))
    end = int(match.group(2))
    print(start, end)
```

Reply:
This is much simpler, see implementation in this commit: https://github.com/alrevuelta/rest-traffic/pull/8/commits/72b62b0c2ae64e9db36755337feb0d944691473e